surfer/utils.py

```diff
@@ -32,7 +32,7 @@ class _MayaviThresholdPatch(object):
     In Mayavi 4.5, filters seem to be missing a .point_data attribute that
     Threshold accesses on initialization.
     """
-    need_patch = LooseVersion(mayavi.__version__) <= LooseVersion('4.5.0')
+    need_patch = LooseVersion(mayavi.__version__) == LooseVersion('4.5.0')
```

Comment:
I should have been more clear in my previous review that I am ignorant about the issue here. When I suggested using `==` I was just saying that was implied by the docstring. Do we know the underlying issue will be fixed in Mayavi 4.5.1?